_posts/2025-02-01-next-scala-lts.md

```diff
@@ -0,0 +1,125 @@
+---
+category: announcement
+permalink: /news/next-scala-lts.html
+title: "Next Scala 3 LTS version will increase minimum required JDK version"
```

Comment:
Maybe we should use `LTS series` instead of `LTS version`. Otherwise just from headline somebody might think that Scala 3.3.6 LTS would require newer JDK.

Reply:
sure, updated!